coreos-installer

```diff
@@ -613,9 +613,23 @@ log() {
 #########################################################
 write_image_to_disk() {
     log "Writing disk image"
-    
+
+    set -o pipefail
+    # Check for gzip and xz correspondingly
+    if [ "$(dd count=2 bs=1 if=/mnt/dl/imagefile.gz)" = "$(printf '\x1f\x8b')" ]
```

Comment:
Minor/optional: maybe we should rename this filename to something more neutral? E.g. maybe just `imagefile`? Or `imagefile.compressed`?

Reply:
Will be updating :ok_hand: